src/lib/graph/node-mapping.ts

```diff
@@ -231,20 +253,14 @@ export const UNIFORM_MAPPING: UniformMapping<nodeTypes.All, util.UniformConfig>
   vector: (node) => [
     {
       type: 'vec3f',
-      key: getHandleKey({
-        sourceNode: node,
-        outgoingHandleId: nodeTypes.HANDLES.vector.out.result,
-      }),
+      key: formatKey(`hdlkey_${node.id}_${nodeTypes.HANDLES.vector.out.result}`),
```

Comment:
ooh i see how this gets complicated if we reuse `getHandleKey`... maybe we can figure out a cleaner way later lol